Add support for LTE v6 collector

SijmenHuizenga opened the pull request:
I tried the new LTE collector on our routers, but it didn't work. Turns out the path on routerOs v6 is a bit different. 

Polluting the `interface_ds.py` with extra conditions specifically for the LTE collector made it all look messy, so instead I opted for creating a specific datasource specifically for LTE.

I am unable to test this on a routerOs v7, as I'm lacking such a device. However I took care to keep the original logical steps the same, so it should (hope) still work on v7 as well.

SijmenHuizenga commented:
An open issue with the LTE collector on v6 is that it requires more permissions. A user with the documented `api,read` permissions will result in the following error:

Error getting lte interface monitor info from router router@10.42.0.1: ('Error "not enough permissions (9)" executing command b\'/interface/lte/info =once= =number=0 .tag=31\'', b'not enough permissions (9)')

akpw commented:
@SijmenHuizenga thanks for contributing. Looks like the change is basically using RouterOSv6 `/interface/lte/info` command vs the routerOS7 `/interface/lte/monitor` menu. Would not it be easier address it in existing code rather than duplicating most of it and adding a dedicated DS? 
Also (and more importantly), what are the additional permissions required to run the OSv6 command? Not sure it'd be desirable to extend the current `api,read` permission scope.

SijmenHuizenga commented:
Found the missing policy after trying them 1-by-1. The `test` policy is required to do the 'lte info' on routeros6

SijmenHuizenga commented:
I simplified the code by adjusting the existing DS and removing the dedicated LTE DS.

I've also added comments about the permissions required on RouterOS 6.

And squashed all the tiny commits that I used for testing into a single nice commit. Works great on here!

akpw commented:
OK looks good
